operations/exports.md

```diff
@@ -0,0 +1,158 @@
+# Exports
```

Comment:
I think this should have a use case described as well for sure 👍 Or we can make it later, when we remove restricted.

Reply:
I believe use case should go on a separate page, let's keep the reference for reference only.